### DISCUSSION TITLE
Cannot access API documentation from Wiki

### DISCUSSION THREAD

**Discussion by Superstellar** [ANLAB-KAIST/KENSv3#67] [Project Q&A 4. Etc]
I cannot access API documentation page from wiki hyperlink, which is:
[API Documentation](http://anlab-kaist.github.io/KENSv3/doc/).

It would be better if it is fixed to (http://anlab-kaist.github.io/KENSv3/doc/index.html).

**Comment by Pusnow** [comment]
Hmm, the original link works for me.
What browser are you using?

**Reply by Superstellar** [reply]
I'm using Chrome browser.

**Reply by Pusnow** [reply]
We've test the page on the Chrome browsers (both Windows and Mac), and it works okay.

This could be a [Github page's issue](https://www.githubstatus.com/incidents/k6vy6ypz04d9) or due to a misconfiguration of your browser.
